hcs-sxc-java-plugins/hcs-sxc-java-plugins-mirror-queue-artemis/pom.xml

```diff
@@ -29,13 +30,14 @@
           <version>${artemis-jms-client-all.version}</version>
       </dependency>
     </dependencies>
-    <build>
+<!--    <build>
```

Comment:
should re-enable through profile at a minimum

Reply:
done